src/MentorBot.Business/Services/HangoutsChatService.cs

```diff
@@ -35,7 +38,9 @@ public HangoutsChatService(ICognitiveService cognitiveService)
                 return new ChatEventResult(Messages.UnknownCommandText);
             }
 
-            var result = await command.CommandProcessor.ProcessCommandAsync(command.TextDeconstructionInformation, chatEvent, null);
+            var result = await command
```

Comment:
I think we can just return it:
`return command`
           `.CommandProcessor`
           `.ProcessCommandAsync(command.TextDeconstructionInformation, chatEvent, _responder);`

Reply:
This is a good option if we only had one async task. We can still do it, but with 2 async tasks it may have some side effects. We can play around with this in the unit tests.